plugins/woocommerce/src/Admin/BlockTemplates/README.md

```diff
@@ -0,0 +1,202 @@
+# BlockTemplates
+
+The `Automattic\WooCommerce\Admin\BlockTemplates` namespace contains interfaces for working with block templates.
+
+## Usage
+
+Instances that implement the interfaces and fire the hooks in ths namespace are created using more specific APIs,
```

Comment:
Small typo 'ths'.

Reply:
I found this 'Usage' section a little vague. Since the title is 'Usage', and this namespace contains only interfaces, maybe you can start by saying that this should not be used directly. Also, I would use the word 'implemented' instead of 'created', since we're talking about Interfaces and Classes.

Reply:
One more comment after going through all the changes: maybe you can also mention that the interfaces from this package will need to be imported in order to use the type declarations.

Reply:
> Small typo 'ths'.

Hrmm... my spell checker did not catch this for some reason?!

Reply:
> Also, I would use the word 'implemented' instead of 'created', since we're talking about Interfaces and Classes.

I'll tweak the wording a bit to make it more clear. But, I do think created is more correct, as I'm referring to the instances of the classes (the objects).